### PR TITLE
Add AdGuard Home

### PR DESCRIPTION
**Pull request by sevospl**

### CODE REVIEW

File: docs/community/apps/adguardhome.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Project Information
+
+- [:material-home: AdGuard Home ](https://adguard.com/en/adguard-home/overview.html){: .header-icons target=_blank rel="noopener noreferrer" }
+- [:octicons-link-16: Docs](https://kb.adguard.com/en/home/overview){: .header-icons target=_blank rel="noopener noreferrer" }
+- [:octicons-mark-github-16: Github:](https://github.com/AdguardTeam/AdGuardHome){: .header-icons target=_blank rel="noopener noreferrer" }
+- [:material-docker: Docker: ](https://hub.docker.com/r/adguard/adguardhome){: .header-icons target=_blank rel="noopener noreferrer" }
+
+!!! info
+    AdGuard Home is a latency sensitie DNS server, so it's discouraged to use it when your server is far away from you.
```
Comment (chazlarson) [inline docs/community/apps/adguardhome.md:15]: Typo: "sensitie"

File: mkdocs.yml [hunk 2]
```diff
@@ -1,299 +1,300 @@
-site_name: docs.saltbox.dev
```
Comment (chazlarson) [inline mkdocs.yml:1]: Why has this entire file changed?

File: docs/community/apps/adguardhome.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+```
+
+### 2. URL
+
+- To access AdGuard Home dashboard, visit `https://adguardhome._yourdomain.com_`
+
+### 3. Usage
+
+- Make sure you have an application that supports DNS over HTTPS, e.g. [Intra for Android](https://play.google.com/store/apps/details?id=app.intra) or [DNSCloak for iOS](https://apps.apple.com/us/app/dnscloak-secure-dns-client/id1452162351)
```
Comment (chazlarson) [inline docs/community/apps/adguardhome.md:31]: Is this only for use with mobile clients?

**Review by chazlarson — COMMENTED** [review]
a few notes.

**Comment by chazlarson** [comment]
This will need to be rebased because mkdocs.yml has changed and the commit really needs to reflect just the changes, not replace the entire file.

**Comment by maximuskowalski** [comment]
Cloned and added to sandbox apps section of docs.

**Pull request closed without merging**